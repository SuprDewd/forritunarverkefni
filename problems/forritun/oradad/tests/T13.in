-51 -2 -84
