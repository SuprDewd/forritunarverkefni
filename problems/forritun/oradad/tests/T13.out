-51 -84 -2
